extensions/google-chrome/src/util/index.ts

```diff
@@ -8,11 +10,21 @@ const userLibraryDirectoryPath = () => {
 
   return path.join(process.env.HOME, "Library");
 };
+const getChromeFilePath = (fileName: string, profile?: string) => {
+  const { profilePath } = getPreferenceValues<Preferences>();
+  if (profilePath) {
+    return path.join(profilePath, fileName);
+  }
+  return path.join(
+    userLibraryDirectoryPath(),
+    ...defaultChromeProfilePath,
+    profile ?? DEFAULT_CHROME_PROFILE_ID,
+    fileName
+  );
+};
```

Comment:
Many bug reports come from an incorrect profile location. Checking whether the file exists here, and explaining what to do if it doesn't, would help prevent those reports.